app/views/includes/foot.jade

```diff
@@ -3,7 +3,8 @@ script.
       (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
       g.src='//www.google-analytics.com/ga.js';
       s.parentNode.insertBefore(g,s)}(document,'script'));
-script(type='text/javascript', src='/lib/jquery/jquery.js')
+//script(type='text/javascript', src='/lib/jquery/jquery.js')
+script(type='text/javascript', src='https://code.jquery.com/jquery-2.2.4.js')
```

Comment:
Why are you using a CDN? When the application is created, it already gets the latest version of jquery in the link you commented above